install-panel.sh

```diff
@@ -28,9 +28,9 @@ set -e
 #                                                                           #
 #############################################################################
 
-# versioning
-GITHUB_SOURCE="master"
-SCRIPT_RELEASE="canary"
+#################################
+######## General checks #########
+#################################
```

Comment:
Are these large comments required? They add three new lines per comment which is quite a lot.

Reply:
But it is a lot easier to find the section you need with them. If you want me to remove them, I can.

Reply:
I think they should be removed.

Reply:
And replaced by anything?

Reply:
One line comments.

Reply:
So from 
```bash
#################################
######## General checks #########
#################################
```
to
```bash
######## General checks #########
```
?